pkg/monitoring/exporter_prometheus_test.go

```diff
@@ -118,6 +118,18 @@ func TestPrometheusExporter(t *testing.T) {
 			chainConfig.GetNetworkID(),     // networkID
 			chainConfig.GetNetworkName(),   // networkName
 		).Once()
+		metrics.On("SetOffchainAggregatorAnswersLatestTimestamp",
+			float64(envelope1.LatestTimestamp.Second()), // latestTimestamp
+			feedConfig.GetID(),                          // contractAddress
+			feedConfig.GetID(),                          // feedID
```

Comment:
why do we have duplicated data

Reply:
This is because for Data Feeds, the feed ID == contract address. In Data Streams, this isn't the case (one contract address can contain many feed IDs).